dotnet/samples/Demos/TelemetryWithAppInsights/Program.cs

```diff
@@ -99,16 +114,17 @@ public static async Task Main()
     /// </summary>
     private static readonly ActivitySource s_activitySource = new("Telemetry.Example");
 
-    private const string AzureOpenAIChatServiceKey = "AzureOpenAIChat";
-    private const string GoogleAIGeminiChatServiceKey = "GoogleAIGeminiChat";
-    private const string HuggingFaceChatServiceKey = "HuggingFaceChat";
+    private const string AzureOpenAIServiceKey = "AzureOpenAI";
+    private const string GoogleAIGeminiServiceKey = "GoogleAIGemini";
+    private const string HuggingFaceServiceKey = "HuggingFace";
```

Comment:
@markwallace-microsoft has a PR out to merge the mistral connector into main. Presumably once that's in we'll want to update it as well with all of this tracing.